### PR TITLE
BXMSDOC-4960 Review and confirm Business Central properties section in the EAP and JWS installation stories

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-4960](https://issues.jboss.org/browse/BXMSDOC-4960)

### CODE REVIEW

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 1]
```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
...
 NOTE: For more information about how to use token based authentication, see xref:usingTokenBasedAuthentication[].
 endif::[]
 
-* `kie.keystore.keyStoreURL`: The URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `\file:///home/kie/keystores/keystore.jceks.`
-* `kie.keystore.keyStorePwd`: The password is used for the JCEKS.
-* `kie.keystore.key.ctrl.alias`: The alias of the key for default REST {CONTROLLER}.
-* `kie.keystore.key.ctrl.pwd`: The password of an alias for default REST {CONTROLLER}.
-* `org.jbpm.wb.forms.renderer.ext`: Switches the form rendering between {CENTRAL} and {KIE_SERVER}. By default, the form rendering is done by {CENTRAL}. Default value: `false`.
-* `org.jbpm.wb.forms.renderer.name`: This property allows to switch between {CENTRAL} and {KIE_SERVER} rendered forms. Default value: `workbench`.
+* `kie.keystore.keyStoreURL`: The URL used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `\file:///home/kie/keystores/keystore.jceks.`
```
Comment (tomasdavidorg) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:86]: `kie.keystore.keyStoreURL` - default value is `""`

https://github.com/kiegroup/kogito-runtimes/blob/master/drools/drools-core/src/main/java/org/drools/core/util/KeyStoreConstants.java#L32

https://github.com/kiegroup/drools/blob/master/drools-core/src/main/java/org/drools/core/util/KeyStoreHelper.java#L118

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 2]
```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
...
-* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver.
-* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!.
-* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
+COMMENT: Default value?
+* `appformer.experimental.features`: Enables the experimental features framework.
+COMMENT: Default value?
+* `org.kie.demo`: Enables an external clone of a demo application from GitHub.
+COMMENT: Default value?
+* `org.kie.workbench.controller`: The URL used to connect to the {CONTROLLER}, for example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
```
Comment (tomasdavidorg) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:77]: `org.kie.workbench.controller` - There is no default property.

https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-api/src/main/java/org/kie/workbench/common/screens/server/management/utils/ControllerUtils.java#L28

https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-api/src/main/java/org/kie/workbench/common/screens/server/management/utils/ControllerUtils.java#L47

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 3]
```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
...
-* `org.kie.demo`: Enables external clone of a demo application from GitHub.
-* `org.kie.workbench.controller`: The URL is used to connect with the {CONTROLLER}. For example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
-* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver.
-* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!.
-* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
+COMMENT: Default value?
+* `appformer.experimental.features`: Enables the experimental features framework.
+COMMENT: Default value?
+* `org.kie.demo`: Enables an external clone of a demo application from GitHub.
```
Comment (tomasdavidorg) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:76]: `org.kie.demo` - There is no default property.

https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/service/ExamplesService.java#L34

https://github.com/kiegroup/jbpm-wb/blob/master/jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseDemoProvisioningService.java#L78

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 4]
```diff
@@ -60,32 +60,39 @@ NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set
...
-* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in {CENTRAL} `.security` folder.
-* `appformer.experimental.features`: Enables the Experimental Features Framework.
-* `org.kie.demo`: Enables external clone of a demo application from GitHub.
-* `org.kie.workbench.controller`: The URL is used to connect with the {CONTROLLER}. For example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
-* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver.
-* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!.
-* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
```
Comment (tomasdavidorg) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:74]: `appformer.ssh.keys.storage.folder` - There is no default property.

https://github.com/kiegroup/appformer/blob/master/uberfire-ssh/uberfire-ssh-backend/src/main/java/org/uberfire/ssh/service/backend/keystore/impl/storage/DefaultSSHKeyStore.java#L40

https://github.com/kiegroup/appformer/blob/master/uberfire-ssh/uberfire-ssh-backend/src/main/java/org/uberfire/ssh/service/backend/keystore/impl/storage/DefaultSSHKeyStore.java#L50

**Review by tomasdavidorg — APPROVED** [review]

**Review by kaldesai — CHANGES_REQUESTED** [review]
Hey @emmurphy1, I reviewed both community and enterprise doc previews. I have a few suggestions. 

1. Following BC properties still reflecting **COMMENT: Default value?** in the enterprise doc previews. 

- appformer.experimental.features
- org.kie.demo
- appformer.ssh.keys.storage.folder
- org.kie.workbench.controller

2. The default value for the following properties is **Default value: ""**. But it is not displaying the latest changes in the enterprise doc previews

- kie.keystore.keyStoreURL
- kie.keystore.keyStorePwd
- kie.keystore.key.ctrl.alias
- kie.keystore.key.ctrl.pwd

3. Drools doc preview is not same as of jBPM doc preview. It is not reflecting the latest changes. Can you please update the link? 
For example, the following section can be removed from drools preview link. 
````
To change one of these system properties in a WildFly or JBoss EAP cluster:

Edit the file $JBOSS_HOME/domain/configuration/host.xml.

Locate the XML elements server that belong to the main-server-group and add a system property, for example:

<system-properties>
  <property name="org.uberfire.nio.git.dir" value="..." boot-time="false"/>
  ...
</system-properties>
````
Apart from this everything looks good to me. Thank you so much for updating and rectifying this content .

**Pull request merged**